perma_web/api/authorizations.py

```diff
@@ -17,7 +17,9 @@ def can_access(self, user, obj):
         # shared folders
         elif obj.vesting_org_id:
             if user.is_vesting_org_member():
-                return user.vesting_org_id == obj.vesting_org_id
+                for vo in user.vesting_org.all():
```

Comment:
Yes. Vesting orgs makes sense. Added an issue so we don't lose track, https://github.com/harvard-lil/perma/issues/873